src/polyfills/template_polyfill.ts

```diff
@@ -22,37 +24,42 @@
  * polyfill: https://github.com/webcomponents/template
  */
 export const initTemplatePolyfill = (forced = false) => {
-  if (typeof HTMLTemplateElement !== 'undefined' && !forced) {
+  if (!forced && 'content' in document.createElement('template')) {
```

Comment:
Can you document `'content' in document.createElement('template')`, specifically mentioning the two versions of lit-html issue.

Reply:
Done.